client/src/global/components/fatal-error/Boundary.js

```diff
@@ -25,13 +25,11 @@ class FatalErrorBoundary extends Component {
   }
 
   componentDidCatch(error, info) {
-    // This will soon be deprecated.
```

Comment:
I took this comment out because I didn't see anything about it being deprecated soon.  Not sure if I missed something or if this was an old comment or something.

Reply:
I think it should be left there. From the React.Component docs: https://reactjs.org/docs/react-component.html

Is getDerivedStateFromError in fact being called? I see the mistake in setting state instead of returning an object, but if memory serves, I was running into problems getting it to call when errors were caught. If that is in fact working, I don't think we need to set the state in `componentDidCatch`.